Mathlib/Order/Filter/Pointwise.lean

```diff
@@ -318,18 +318,27 @@ theorem mul_mem_mul : s ∈ f → t ∈ g → s * t ∈ f * g :=
 #align filter.mul_mem_mul Filter.mul_mem_mul
 #align filter.add_mem_add Filter.add_mem_add
 
-@[to_additive (attr := simp)]
 theorem bot_mul : ⊥ * g = ⊥ :=
   map₂_bot_left
 #align filter.bot_mul Filter.bot_mul
-#align filter.bot_add Filter.bot_add
 
-@[to_additive (attr := simp)]
 theorem mul_bot : f * ⊥ = ⊥ :=
   map₂_bot_right
 #align filter.mul_bot Filter.mul_bot
```

Comment:
These lemmas should stay simp, right?

Reply:
Yes, I make them `simp` later with
```
protected theorem bot_add {α : Type*} [Add α] {g : Filter α} : ⊥ + g = ⊥ := by simp

protected theorem add_bot {α : Type*} [Add α] {f : Filter α} : f + ⊥ = ⊥ := by simp

attribute [to_additive existing] bot_mul mul_bot
attribute [simp] bot_mul mul_bot
```
I did this to avoid creating a collision with the usual `add_top` and `top_add` while still letting `to_additive` work on them.